c-vararg is an internal header of kpathsea, not installed by the system version.

Index: xdvik-22.84.16/texk/xdvik/dvi-draw.c
===================================================================
--- xdvik-22.84.16.orig/texk/xdvik/dvi-draw.c
+++ xdvik-22.84.16/texk/xdvik/dvi-draw.c
@@ -43,7 +43,6 @@ in xdvi.c.
 #include "kpathsea/c-stat.h"
 #include "kpathsea/magstep.h"
 #include "kpathsea/tex-file.h"
-#include "kpathsea/c-vararg.h"
 
 #include <string.h>
 
Index: xdvik-22.84.16/texk/xdvik/gui/message-window.c
===================================================================
--- xdvik-22.84.16.orig/texk/xdvik/gui/message-window.c
+++ xdvik-22.84.16/texk/xdvik/gui/message-window.c
@@ -78,7 +78,6 @@ by the user (as a rough guide: a delay o
 # include <X11/Xaw/Dialog.h>
 #endif
 
-#include "kpathsea/c-vararg.h"
 #include "xdvi.h"
 #include "util.h"
 #include "string-utils.h"
Index: xdvik-22.84.16/texk/xdvik/gui/statusline.c
===================================================================
--- xdvik-22.84.16.orig/texk/xdvik/gui/statusline.c
+++ xdvik-22.84.16/texk/xdvik/gui/statusline.c
@@ -32,7 +32,6 @@
 #include "pagehist.h"
 #include "util.h"
 
-#include "kpathsea/c-vararg.h"
 #include "my-vsnprintf.h"
 
 #include <ctype.h>
Index: xdvik-22.84.16/texk/xdvik/string-utils.h
===================================================================
--- xdvik-22.84.16.orig/texk/xdvik/string-utils.h
+++ xdvik-22.84.16/texk/xdvik/string-utils.h
@@ -2,7 +2,6 @@
 #define STRING_UTILS_H_
 
 #include "xdvi-config.h"
-#include "kpathsea/c-vararg.h"
 #include "my-vsnprintf.h"
 
 extern Boolean str_is_prefix(const char *, const char *, Boolean case_sensitive);
Index: xdvik-22.84.16/texk/xdvik/tests/test_string_utils.c
===================================================================
--- xdvik-22.84.16.orig/texk/xdvik/tests/test_string_utils.c
+++ xdvik-22.84.16/texk/xdvik/tests/test_string_utils.c
@@ -1,6 +1,5 @@
 #include "xdvi-config.h"
 #include "xdvi.h"
-#include "kpathsea/c-vararg.h"
 
 #include <stdio.h>
 #include <stdlib.h>
Index: xdvik-22.84.16/texk/xdvik/xdvi.h
===================================================================
--- xdvik-22.84.16.orig/texk/xdvik/xdvi.h
+++ xdvik-22.84.16/texk/xdvik/xdvi.h
@@ -261,7 +261,6 @@ extern KPSEDLL char *kpathsea_version_st
 #  define NeedVarargsPrototypes	NeedFunctionPrototypes
 #endif
 
-#include "kpathsea/c-vararg.h"
 #include "kpathsea/c-fopen.h"
 
 #ifndef	_XFUNCPROTOBEGIN
